cmake/harfbuzzConfig.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/harfbuzzTargets.cmake")
```

Comment:
`check_required_components` should be used here to check if user mistakenly specify some `COMPONENTS` in `find_package(harfbuzz ....)`:
* https://github.com/cgold-examples/fruits/blob/b266e5a8b0dccc250390346acde07e613562d85d/cmake/template/Config.cmake.in#L4

Reply:
Also we need `find_package` calls here for freetype/ICU dependencies when they used, example:
* https://github.com/elucideye/drishti/blob/9b11f52b8ce0fa93794f3afb7bcdbd0cfa518bc3/cmake/Config.cmake.in#L38-L49